rosa_getting_started/rosa-getting-started.adoc

```diff
@@ -33,7 +33,7 @@ include::modules/rosa-getting-started-install-configure-cli-tools.adoc[leveloffs
 [id="rosa-getting-started-creating-a-cluster"]
 == Creating a ROSA cluster with STS
 
-Choose from one of the following methods to deploy a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS). In each scenario, you can deploy your cluster by using {cluster-manager-first} or the ROSA CLI (`rosa`):
+Choose from one of the following methods to deploy a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS). In each scenario, you can deploy your cluster by using {cluster-manager-first} or the {product-title} (ROSA) CLI (`rosa`):
```

Comment:
only expand product-title once, just ROSA CLI here is fine

Reply:
Maybe , "ROSA CLI (`rosa`)" here?